syntax = "proto3";

package v1;

option go_package = "wjb-dev/demo-go-server/proto/v1;v1";


service EchoService {
  rpc Echo(EchoRequest) returns (EchoResponse);
}

message EchoRequest {
  string message = 1;
}

message EchoResponse {
  string message = 1;
}